docker-images/kafka/scripts/zookeeper_run.sh

```diff
@@ -35,7 +36,8 @@ fi
 
 # enabling Prometheus JMX exporter as Java agent
 if [ "$ZOOKEEPER_METRICS_ENABLED" = "true" ]; then
-  export KAFKA_OPTS="-javaagent:$(ls $KAFKA_HOME/libs/jmx_prometheus_javaagent*.jar)=9404:$KAFKA_HOME/custom-config/metrics-config.yml"
+  KAFKA_OPTS="-javaagent:$(ls $KAFKA_HOME/libs/jmx_prometheus_javaagent*.jar)=9404:$KAFKA_HOME/custom-config/metrics-config.yml"
```

Comment:
Why change working code with what seems to be just a reformating?

Reply:
TBH this was a suggestion from an over zealous Bash linting script that didn't like export and assignment in the same command as it can mask return values. But I can change it back.